docs/openapi/schemas.yaml

```diff
@@ -37,6 +37,14 @@ Domain:
   type: string
   format: domain
   example: 'www.example.com'
+ScopeName:
+  type: string
+  enum:
+    - 'imports.write'
+    - 'imports.read'
+    - 'imports.all_domains'
+    - 'imports.read_all'
+    - 'imports.delete'
```

Comment:
Do we need to call out these 3 in the docs?

Reply:
I could remove it. As it's only for special users.